jQuery/honeyHeadquarters.js

```diff
@@ -1,25 +1,25 @@
 // $.ajax('https://sheetsu.com/apis/v1.0/64b5c3f8').done(function(data) {
 $.ajax('http://localhost:3000/citizens').done(function(citizens) {
```

Comment:
Move the entire block, because we're eventually going to put the functions in their own file as well.